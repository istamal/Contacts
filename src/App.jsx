import React from 'react';
import './App.scss';
import 'antd/dist/antd.css';
import styled from 'styled-components';
import Contacts from './Components/Contacts';

function App() {
  return (
    <Contacts />
  );
}

export default App;
