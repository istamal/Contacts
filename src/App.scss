body {
	background: linear-gradient(45deg, #A798B3, #A9A9A9);
	padding-bottom: 60px;
}

.set-contact {
	margin-right: 20px !important;
}

.margin-right {
	margin-right: 10px !important;
}

.margin-bottom {
	margin-bottom: 10px !important;
}