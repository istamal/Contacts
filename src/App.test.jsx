import React from 'react';
import ReactDOM from 'react-dom';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from './App';

test('renders learn react link', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

test('renders element correctly', () => {
  const { getByTestId } = render(<App />);
  expect(getByTestId('wrapper')).toHaveTextContent('Hello, Istamal!');
});

test('matches snapshot', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
