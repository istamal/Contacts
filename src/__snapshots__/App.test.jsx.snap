// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="sc-AxjAm fAPiVx"
  data-testid="wrapper"
>
  <h1
    className="sc-AxirZ ficMlO"
  >
    Hello, Istamal!
  </h1>
  <p
    className="sc-AxiKw Wwxzi"
  >
    the cra template
  </p>
  <p
    className="sc-AxhUy bgbkIn"
  >
    The template includes
     
    <span
      className="sc-AxhCb bmEVDm"
    >
      Antd
    </span>
    ,
    <span
      className="sc-AxhCb bmEVDm"
    >
      Prop-Types
    </span>
    ,
    <span
      className="sc-AxhCb bmEVDm"
    >
      Styled-Components
    </span>
    .
    <span
      className="sc-AxhCb bmEVDm"
    >
      Node-Sass
    </span>
     
    and more.
  </p>
</div>
`;
